docs/API/Arguments.md

```diff
@@ -43,6 +42,16 @@ Use `window.__REDUX_DEVTOOLS_EXTENSION__([config])` or `window.__REDUX_DEVTOOLS_
        }
      }));
      ```
+    - **immutable** `object`: Automatically serialize/deserialize immutablejs via [remotedev-serialize](https://github.com/zalmoxisus/remotedev-serialize)
+    ```js
+        // Like above, only showing off compose this time. Reminder you might not want this in prod.
+        const composeEnhancers = typeof window === 'object' && typeof window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ !== 'undefined' ?
+                                 window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
+                                     serialize: {
+                                         immutable: require('immutable') // https://facebook.github.io/immutable-js/
```

Comment:
I guess better to use `require('immutable').default`, as the above would require `babel-plugin-add-module-exports` to get it work, which is not widely used.

Even better to use `import immutable from 'immutable';` as it will be included in the app anyway, so we'll avoid confusion and including it twice.